scripts/menu.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+$(function() {
```

Comment:
It is great this is showing the nav on click. I want to challenge you to make it so that the hamburger menu will come back when you click on an x or another part of the nav or when you resize!